### PR TITLE
add validation for revision name length

### LINKED ISSUE #40: [Feature request] Revision name (not only suffix) length validaton
The max length of revision name ( the format is `${CONTAINER_APP_NAME}--${SUFFIX}`) seems to be limited to 63.
In case that the length is tried to set more then 63, the revision is not added but the REST API/SDK don't return error unfortunately.

So, it's needed to check if the revition is included in revision list of a container app after adding the revision, or to validate length of both of inputs `name` and `revision-name-suffix` value in this action.

To check `latestRevisionFqdn` after adding the revision might not be valid with considering concurrent updates.

### PR DESCRIPTION
**Pull request by horihiro**
To resolve #40 
- add check logic for validating revision name length

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -20,7 +20,9 @@ async function main() {
 
     let endpoint: IAuthorizer = await AuthorizerFactory.getAuthorizer();
     var taskParams = TaskParameters.getTaskParams(endpoint);
-    let credential: TokenCredential = new DefaultAzureCredential()
+    let credential: TokenCredential = new DefaultAzureCredential();
+    const revisionName = `${taskParams.containerAppName}--${taskParams.revisionNameSuffix}`;
```
Comment (koudaiii) [inline src/main.ts:24]: Why is there a `--`? Reference docs or comments would be appreciated.
Can we put `--` in revisionNameSuffix?

Comment (horihiro) [reply src/main.ts:24]: This is documentation describing the format.
https://learn.microsoft.com/en-us/azure/container-apps/revisions#revision-name-suffix
image

Comment (koudaiii) [reply src/main.ts:24]: Would you like to comment on this URL link about this magic symbol? I think we should know this specification.

Comment (horihiro) [reply src/main.ts:26]: Added.

**Review by koudaiii — APPROVED** [review]
LG 👍

**Pull request merged**